ocfweb/api/urls.py

```diff
@@ -3,10 +3,13 @@
 from ocfweb.api import hours
 from ocfweb.api import lab
 from ocfweb.api import session_tracking
+from ocfweb.api import shorturls
+
 
 urlpatterns = [
     path('hours', hours.get_hours_all, name='hours_all'),
     path('hours/today', hours.get_hours_today, name='hours_today'),
     path('lab/desktops', lab.desktop_usage, name='desktop_usage'),
     path('session/log', session_tracking.log_session, name='log_session'),
+    path('shorturl/<slug:slug>', shorturls.bounce_shorturl, name='bounce_shorturl'),
```

Comment:
We don't actually have a slug path converter, do we? Shouldn't this just be `'shorturl/<slug>'`?

Reference: https://docs.djangoproject.com/en/2.1/topics/http/urls/#registering-custom-path-converters

Reply:
it's built-in, I think. This seems to work fine when I run it in testing.